code/Kotoba/AppDelegate.swift

```diff
@@ -30,9 +26,15 @@ class AppDelegate: UIResponder, UIApplicationDelegate
 		}
 		
 		debugLog("libraryDirectory = \(NSSearchPathForDirectoriesInDomains(.libraryDirectory, .userDomainMask, true))")
-		
-		// Set tint colour to match icon
-		UIView.appearance().tintColor = _RED_THEME_COLOUR
```

Comment:
This is now applied [at the root view](https://github.com/willhains/Kotoba/pull/85/files#diff-38a309b36604feb582084669cdb9e410aacdc9e74e98b452a4264518fb3220a6R167) and should propagate to the rest of the app.